aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -193,6 +193,63 @@ void qrelu6_kernel(const Tensor& qx, Tensor& qy) {
   });
 }
 
+void qsigmoid_kernel(const Tensor& qx, Tensor& qy) {
+  int64_t zero_point = qx.q_zero_point();
+  float scale = qx.q_scale();
+  float inv_scale = 1.0f / scale;
+  auto scale_vec = Vec256<float>(scale);
+  auto zero_point_vec = Vec256<float>((float)zero_point);
+  auto scale_neg_zp_premul_vec = scale_vec * zero_point_vec.neg();
+
+  AT_DISPATCH_QINT_TYPES(qx.scalar_type(), "qsigmoid", [&]() {
```

Comment:
Can you also add benchmark results showing time spent on different operations within an LSTM cell to understand what fraction is spent in sigmoid?

Reply:
That will be a separate set of PRs during the optimization